.github/workflows/compose-test.yml

```diff
@@ -0,0 +1,126 @@
+  pull_request:
+
+jobs:
+  run-docker-compose:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Set up Docker Compose
+        run: |
+          sudo apt-get update
+          sudo apt-get install docker-compose -y
+
+      - name: Build indexer container
+        uses: docker/build-push-action@v5
+        with:
+          load: true
+          context: .
+          file: ./indexer/Dockerfile
+          tags: near-sffl-indexer
+          cache-from: type=gha,scope=indexer
+          cache-to: type=gha,mode=max,scope=indexer
+
+      - name: Build relayer container
+        uses: docker/build-push-action@v5
+        with:
+          load: true
+          context: .
+          file: ./relayer/cmd/Dockerfile
+          tags: near-sffl-test-relayer
+          cache-from: type=gha,scope=relayer
+          cache-to: type=gha,mode=max,scope=relayer
+
+      - name: Build aggregator container
+        uses: docker/build-push-action@v5
+        with:
+          load: true
+          context: .
+          file: ./aggregator/cmd/Dockerfile
+          tags: near-sffl-aggregator
+          cache-from: type=gha,scope=aggregator
+          cache-to: type=gha,mode=max,scope=aggregator
+
+      - name: Build operator container
+        uses: docker/build-push-action@v5
+        with:
+          load: true
+          context: .
+          file: ./operator/cmd/Dockerfile
+          tags: near-sffl-operator
+          cache-from: type=gha,scope=operator
+          cache-to: type=gha,mode=max,scope=operator
+
+      - name: Build and start Docker Compose services
+        run: docker-compose -f ./docker-compose.yml up --build -d
+
+      - name: Wait for 5 minutes
+        run: sleep 300
+
+      - name: List running containers
+        run: |
+          echo "Running containers:"
+          docker ps --format "{{.Names}}"
+
+      - name: Ensure containers are running
+        run: |
+          containers=(
+            "near-sffl_operator1-health_1"
+            "near-sffl-operator1"
+            "near-sffl_operator0-health_1"
+            "near-sffl-operator0"
+            "rollup1-relayer"
+            "rollup0-relayer"
+            "near-sffl-indexer"
+            "near-sffl-aggregator"
+            "mainnet-anvil"
+            "prometheus"
+            "rollup0-anvil"
+            "rmq"
+            "rollup1-anvil"
+            "grafana"
+          ) # List of container names
+          for container in "${containers[@]}"; do
+            if docker inspect -f '{{.State.Running}}' "$container" >/dev/null 2>&1; then
+              echo "$container is running"
+            else
+              echo "$container is not running"
+              exit 1
+            fi
+          done
+
+
+      - name: Check health status
+        run: |
+          services=(
+            "near-sffl_operator1-health_1"
```

Comment:
Is the intention here to check health of `operator` or `operator1-health` image? Because the latter is done. As I remember operator1-health will be running even if operator crashes

Reply:
Discussed in call, `operator` should be checked for running state, and `operator-health` for healthiness until #240 is solved